### ISSUE TITLE
Banned.

### ISSUE THREAD

**Issue by theasdaf67** [Citrinate/gleamSolver#12]
I got banned. "Sorry, you're creating too many entries; try again later." needs update. :

**Comment by Citrinate** [comment]
Maybe more will chime in, but people have been reporting bans for a few weeks now, although very infrequently.  Presently I don't know what's causing them.

Realistically, if Gleam really wanted to detect this script, there's nothing stopping them.  I've personally switched over to [Gleam.helper](https://github.com/Citrinate/gleamHelper), and I'm not sure if I'm going to be supporting this script for much longer.

**Comment by adebisi** [comment]
are the people getting banned using multiple accounts?

**Comment by Vowwl** [comment]
I have been using this script with my alternative accounts and I haven't been banned for ages. It's sad that this script isn't being maintained anymore.

**Comment by Wrascon** [comment]
Of course. I did so many things to avoid it: attached to each account Gleam, personal Facebook, Twitter, Twitch, and certainly Steam. All this worked as long until he became interested in such scripts 3 months ago. Now I'm working 3/8 of accounts, but recently has blocked Twitter and others, and then I wrote sms recovery exceeded the amount of use or something similar. So I completely fucked. 
Of course. I did so many things to avoid it: attached to each account Gleam, personal Facebook, Twitter, Twitch, and certainly Steam. All this worked as long until he became interested in such scripts 3 months ago. Now I'm working 3/8 of accounts, but recently has blocked Twitter and others, and then I wrote sms recovery exceeded the amount of use or something similar. So I completely fucked.

**Comment by Wrascon** [comment]
And yes, Twitter ban I received after the first day Gleam.helper use. It fucking blew me out of the rut of manual farming. I even been preserved accounts as follows, in order not to switch to other accounts as I perform other tasks. Just as it was when I had just started the on first multi-client Gleam. I think because of that I have for a long time can not farm keys as before. Do not use anything like that. Successful development, goodbye

**Comment by RoseTheFlower** [comment]
I had only one account and got banned when using your script version 1.5.3. I've got tens of gifts by using the script before it happened though.

**Comment by nightwind209** [comment]
I use this and banned with ONE new account in 4 days.
I think Gleam detect this script already.
i still thanks author for this, it save many my time.
thanks :)
